src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -135,7 +136,7 @@ export default ({
       forceMobileNoMargin={true}
     >
       <div className="box-insights">
-        <p dangerouslySetInnerHTML={{ __html: messageModal ?? t('insightsText', { socialNetwork: 'instagram' }) }} />
+        <p dangerouslySetInnerHTML={{ __html: messageModal || t('insightsText', { socialNetwork: 'instagram' }) }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)